dex/testing/dcr/harness.sh

```diff
@@ -277,6 +277,11 @@ if [ "$MINE" = "1" ]; then
   done
 fi
 
+# Hack to enable further mining on alpha or beta
+tmux send-keys -t $SESSION:0 "./mine-beta 1${WAIT}" C-m\; wait-for donedcr
+tmux send-keys -t $SESSION:0 "./beta addnode 127.0.0.1:${ALPHA_NODE_PORT} remove${WAIT}" C-m\; wait-for donedcr
+tmux send-keys -t $SESSION:0 "./beta addnode 127.0.0.1:${ALPHA_NODE_PORT} add${WAIT}" C-m\; wait-for donedcr
```

Comment:
This workaround should not be needed with https://github.com/decred/dcrd/pull/2513 merged in dcrd.  Can you verify that you still require this workaround?  If so, we should investigate the issue and see if there's a new bug.

Reply:
Right. I was gonna highlight this after submitting the PR to confirm if it's a known issue with a known fix or workaround but it skipped my mind last minute.

I've updated dcrd to master (commit `8c289356a7e3ef0106627818601cab8acea04e49`) and there seem to be a new issue. Occurs after generating 2 addresses for each wallet; when trying to send 24 DCR to the wallets:
https://github.com/decred/dcrdex/blob/14f6b927d7c162333c84ef5ca588eaebf2f564bc/dex/testing/dcr/harness.sh#L281-L284

#### Screenshot
<img width="958" alt="Screenshot 2020-12-30 at 18 50 10" src="https://user-images.githubusercontent.com/18400051/103371723-816d2980-4ad0-11eb-85c9-efde6576593f.png">

Will investigate further later but dropping this here in case anyone else had encountered this before or knows off the bat what the issue is.

Reply:
Works fine with commit a17427395603bc58cb461a7555a125d65a2e3334 (i.e. https://github.com/decred/dcrd/pull/2513). The issue I noted above is with latest dcrd/master which has its rpc version bumped to 7.0.0. while dcrwallet/master requires 6.0.0.

This workaround is not needed. Removing.